Project.toml

```diff
@@ -8,6 +8,7 @@ AbstractFFTs = "621f4979-c628-5d54-868e-fcf4e3e8185c"
 BlockArrays = "8e7c35d0-a365-5155-bbbb-fb81a777f24e"
 Conda = "8f4d0f93-b110-5947-807f-2305c1781a2d"
 FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
+FiniteDiff = "6a86dc24-6348-571c-b903-95158fe2bd41"
```

Comment:
Can you add this to [extras] instead? That's where we put the "optional" deps used in tests